app/src/constants/colours.ts

```diff
@@ -65,6 +65,16 @@ const NRM_REGION_COLOUR_MAP = {
   'Skeena Natural Resource Region': { colour: red }
 };
 
+/**
+ * Colour map for access request chips.
+ *
+ */
+const ACCESS_REQUEST_STATUS_COLOUR_MAP = {
```

Comment:
praise: Hell yeah! 
This is the correct way for these types of values, instant lookup vs having to traverse through an array of values.